onnx/reference/ops/op_cast.py

```diff
@@ -13,26 +13,29 @@
     float8e4m3fnuz,
     float8e5m2,
     float8e5m2fnuz,
+    float8e8m0,
     int4,
     uint4,
 )
 from onnx.helper import (
     float32_to_bfloat16,
     float32_to_float8e4m3,
     float32_to_float8e5m2,
+    float32_to_float8e8m0,
     tensor_dtype_to_np_dtype,
 )
 from onnx.numpy_helper import (
     bfloat16_to_float32,
     float8e4m3_to_float32,
     float8e5m2_to_float32,
+    float8e8m0_to_float32,
     unpacked_float4e2m1_to_float32,
 )
 from onnx.onnx_pb import TensorProto
 from onnx.reference.op_run import OpRun
 
 
-def cast_to(x, to, saturate):  # noqa: PLR0911
+def cast_to(x, to, saturate, round_mode="up"):  # noqa: PLR0911
```

Comment:
Would should simplify this function with ml_dtypes. I will create a proposal separately

Reply:
You can now import and use ml_dtypes